### PR TITLE
Do not indent where clause by default (follow standard) #257

### PR DESCRIPTION
**Pull request by sebastiencs**

### CODE REVIEW

**Comment by tromey** [comment]
Looks like the tests need updating as well.

**Comment by sebastiencs** [comment]
Oups I haven't checked

File: rust-mode-tests.el [hunk 1]
```diff
@@ -451,6 +451,7 @@ fn foo4(a:i32,
 "))
 
 (ert-deftest indent-body-after-where ()
+  (let ((rust-indent-where-clause t))
   (test-indent
```
Comment (tromey) [inline rust-mode-tests.el]: This is fine but would you mind reindenting the bodies of the `let`s?

**Comment by sebastiencs** [comment]
Alright, I have squashed the commits

**Comment by tromey** [comment]
Maybe I misunderstood what github was showing.  If so, my apologies.  Anyway, thank you very much.

**Comment by sebastiencs** [comment]
No you were correct, I have reindented the `let`s before squashing :)

**Pull request merged**